pandas/tests/test_compat.py

```diff
@@ -8,66 +8,51 @@
 import pytest
 
 from pandas.compat import (
-    builtins, filter, get_range_parameters, iteritems, iterkeys, itervalues,
-    lfilter, lmap, lrange, lzip, map, next, range, re_type, zip)
+    builtins, get_range_parameters, iteritems, iterkeys, itervalues, lfilter,
+    lmap, lrange, lzip, re_type)
 
 
 class TestBuiltinIterators(object):
 
     @classmethod
-    def check_result(cls, actual, expected, lengths):
-        for (iter_res, list_res), exp, length in zip(actual, expected,
-                                                     lengths):
-            assert not isinstance(iter_res, list)
-            assert isinstance(list_res, list)
-
-            iter_res = list(iter_res)
-
-            assert len(list_res) == length
-            assert len(iter_res) == length
-            assert iter_res == exp
-            assert list_res == exp
-
-    def test_range(self):
```

Comment:
This is pretty much the only non-trivial change - edited some tests that would now just be testing the builtin `range`/`map`/`filter`/`zip` functions.  Updated to just test the list equivalent functions, and slightly changed some variable names to better match other test conventions.